Makes the udev rules run Brltty through Upstart instead of just
running it directly.
diff --git a/Autostart/Udev/rules b/Autostart/Udev/rules
index 3b6edc70e..d524aefaf 100644
--- a/Autostart/Udev/rules
+++ b/Autostart/Udev/rules
@@ -464,19 +464,18 @@ ENV{BRLTTY_BRAILLE_DEVICE}="usb:vendor=0X$sysfs{idVendor}+product=0X$sysfs{idPro
 GOTO="brltty_run"
 
 LABEL="brltty_run"
-TEST=="/sys/fs/cgroup/systemd", GOTO="brltty_systemd"
-ENV{BRLTTY_PID_FILE}="/run/brltty.$kernel.pid"
+ENV{BRLTTY_PID_FILE}="/run/brltty/brltty.$kernel.pid"
 
-ACTION=="add", GOTO="brltty_add"
+ACTION=="add|change", GOTO="brltty_add"
 ACTION=="remove", GOTO="brltty_remove"
 GOTO="brltty_end"
 
 LABEL="brltty_add"
-RUN+="brltty-wrapper"
+RUN+="/sbin/start brltty"
 GOTO="brltty_end"
 
 LABEL="brltty_remove"
-RUN+="brltty-wrapper -C"
+RUN+="/sbin/stop brltty"
 GOTO="brltty_end"
 
 LABEL="brltty_systemd"
